orca-api-tck/src/main/resources/orca-test-app.yml

```diff
@@ -29,3 +29,7 @@ redis:
 spring:
   application:
     name: orca
+
+spinnaker:
+  extensibility:
+    plugins-root-path: build/plugins
```

Comment:
I currently have test configuration in the plugin repo as such:

```
test {
  libsDirName = 'plugins'
  dependsOn("assemblePlugin")
}
```
I'll probably move something like that to our Grardle plugin test configuration so developers don't need to do this.